SOURCE/FlowLaws.c

```diff
@@ -894,3 +894,30 @@ void ReadDataGSE( mat_prop* mat, params* model, int k, int number, scale* scalin
 /*--------------------------------------------------------------------------------------------------------------------*/
 /*------------------------------------------------------ M-Doodz -----------------------------------------------------*/
 /*--------------------------------------------------------------------------------------------------------------------*/
+
+void ReadDataKinetics( mat_prop* mat, params* model, int k, int number, scale* scaling ) {
+    
+    int success=0;
```

Comment:
I don't understand why do we need this

Reply:
We need this in case we enter several kinetic law (then each law has its own ID)